tokio-macros/src/entry.rs

```diff
@@ -128,17 +127,28 @@ fn parse_knobs(
         }
     }
 
-    let mut rt = quote! { tokio::runtime::Builder::new().basic_scheduler() };
+    let mut rt = quote! { tokio::runtime::Builder::new() };
+
     if rt_threaded && (runtime == Some(Runtime::Threaded) || (runtime.is_none() && !is_test)) {
-        rt = quote! { #rt.threaded_scheduler() };
-    }
-    if let Some(v) = core_threads.map(|v| v.get()) {
-        rt = quote! { #rt.core_threads(#v) };
+        rt = quote! { #rt };
     }
-    if let Some(v) = max_threads.map(|v| v.get()) {
-        rt = quote! { #rt.max_threads(#v) };
+
+    // if let Some(v) = core_threads {
+    //     rt = quote! { #rt.core_threads(#v) };
+    // }
+
+    // if let Some(v) = max_threads {
+    //     rt = quote! { #rt.max_threads(#v) };
+    // }
+
+    if is_test {
+        rt = quote! { #rt.core_threads(0) };
     }
 
+    // if is_test && core_threads.is_none() {
+    //     rt = quote! { #rt.core_threads(0) };
+    // }
```

Comment:
Don't really care if we merge this, likely going to be rewritten in a follow up PR anyways.

Reply:
This doesn't include #2906, so yes, it will be rewritten by that PR if this is merged first.